test/io/configs/expected/no-defaults-with-db-other-authenticator.config

```diff
@@ -35,6 +35,7 @@ server-trace-header = "traceparent"
 server-timing-enabled = true
 server-unix-socket = "/tmp/pgrst_io_test.sock"
 server-unix-socket-mode = "777"
+admin-server-host = "127.0.0.1"
```

Comment:
Btw, I also pushed a commit that changed the `no-default` tests. Since it uses it as default, `admin-server-host` must be different from `server-host`.

Do a `git pull` before making any changes.

Reply:
Thank you!